desktop/build.gradle

```diff
@@ -24,20 +24,17 @@ compileJava {
 
 repositories {
     mavenCentral()
-
-    maven {
-        url  "http://dl.bintray.com/ijabz/maven"
-    }
+    maven { url "https://dl.bintray.com/ijabz/maven/" }
 }
 
 dependencies {
-    compile 'com.googlecode.gettext-commons:gettext-commons:0.9.8'
-    compile "com.h2database:h2:1.4.193"
-    compile 'org.apache.lucene:lucene-core:3.6.2'
-    compile 'net.jthink:jaudiotagger:2.2.5'
-    compile "com.frostwire:jlibtorrent:1.2.0.18"
+    implementation fileTree(dir: 'lib/jars', include: ['*.jar'])
 
-    compile fileTree(dir: 'lib/jars', include: ['*.jar'])
+    implementation 'com.googlecode.gettext-commons:gettext-commons:0.9.8'
+    implementation 'com.h2database:h2:1.4.197'
```

Comment:
requires testing, and perhaps even migration code

Reply:
I wish more projects used semantic versioning... I figured just going from 1.4.193 to 1.4.197 wouldn't cause any breakage, but apparently it did. Sorry.